qiskit_nature/results/vibrational_structure_result.py

```diff
@@ -69,15 +69,34 @@ def __str__(self) -> str:
     def formatted(self) -> List[str]:
         """Formatted result as a list of strings"""
         lines = []
-        lines.append("=== GROUND STATE ENERGY ===")
+        lines.append("=== GROUND STATE ===")
         lines.append(" ")
         lines.append(
             "* Vibrational ground state energy "
             f"(cm^-1): {np.round(self.computed_vibrational_energies[0], 12)}"
         )
         if len(self.num_occupied_modals_per_mode) > 0:
             lines.append("The number of occupied modals is")
```

Comment:
Maybe we can improve this a little bit, too:
```suggestion
            lines.append("The number of occupied modals for each mode is: ")
```